x-pack/plugins/cloud_security_posture/README.md

```diff
@@ -4,6 +4,84 @@ Cloud Posture automates the identification and remediation of risks across cloud
 
 ---
 
+## Table of contents
+
+- [Development](#development)
+  - [Install pre-commit hooks](#install-pre-commit-hooks)
+  - [Local checks before creating a PR](#local-checks-before-creating-a-pr)
+  - [Running unit tests](#running-unit-tests)
+  - [Running integration tests](#running-integration-tests)
+
+---
```

Comment:
i don't think we need to define this, we should be able to get this for [free](https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/)

Reply:
Nice comment, I wasn't aware of this feature.

I tried to seek out how this feature work, and it doesn't generate a table of context as explicit as the current code suggested.
It lets you use some mini-menu sidebar to select the relevant topic you wish to go to. 

You can check my git commit and see how it looks.

https://github.com/elastic/kibana/tree/a657024e820c39d0e914f8465cfd0c2dc533aacf/x-pack/plugins/cloud_security_posture

<img width="1146" alt="image" src="https://user-images.githubusercontent.com/19221736/188489023-632e9d19-b519-4594-ba5f-36eb3b68ebc1.png">

If it's really important to you, and you think it's a must - I can remove it. 
It is aligned with many other public repositories and personally, I find it easier to navigate using it.

Reply:
i'm fine with both, it's just that using the generated one keeps us from keeping those links updated. that's it.. hardly an issue tho